proto/api/v1/auth_service.proto

```diff
@@ -20,21 +21,149 @@ message UserAttribute {
 
 message AuthStatus {
     oneof id {
-        string                user_id     = 1;
-        storage.ServiceIdentity       service_id  = 2;
+        string                  user_id         = 1;
+        storage.ServiceIdentity service_id      = 2;
     }
-    google.protobuf.Timestamp expires     = 3;
-    string                    refresh_url = 4;
+    google.protobuf.Timestamp   expires         = 3;
```

Comment:
`expiry`

Reply:
This is an existing API proto, it cannot be simply renamed.

Also, this is not in scope of this PR, it merely is fixing formatting. I'm not willing to make any changes to this existing API in this PR.